test/test_serialization.py

```diff
@@ -0,0 +1,82 @@
+import contextlib
+import os.path
+import pickle
+from pathlib import Path
+
+from exasol_script_languages_developer_sandbox.lib.cf_stack import CloudformationStack
+from exasol_script_languages_developer_sandbox.lib.key_file_manager import KeyFileManager
+from test.aws_local_stack_access import AwsLocalStackAccess
+import multiprocessing as mp
+
+
+def create_key_pair_and_serialize(tmp_location: Path, q: mp.Queue):
```

Comment:
wrap content maybe in try except with print exception, because otherwise you don't get any hint what is going wrong